Jenkinsfile

```diff
@@ -201,792 +98,78 @@ pipeline {
        }
        stage('Unit Tests') {
            parallel {
-                stage('1 Ubuntu 18.04') {
+                stage('Ubuntu 20.04') {
                     agent {
-                        label "bionic-agent"
+                        label "ubuntu2004-agent"
                     }
                     steps {
                         catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-                            unstash "bionic-unit-tests"
-                            dir ('test') {
-                                sh "./carta_backend_tests --gtest_output=xml:ubuntu_bionic_test_detail.xml"
+                            dir ('build/test') {
+                                sh "echo 'leak:*casacore*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*casa*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*libcurl*' >> ../../debug/asan/myasan-leaks.supp && echo 'leak:*libwcs*' >> ../../debug/asan/myasan-leaks.supp"
```

Comment:
Better create another file `debug/asan/myasan-leaks.supp` and keep those records permanently there.